mne_bids/mne_bids.py

```diff
@@ -619,6 +644,33 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
                                      output_path=output_path, make_dir=False,
                                      overwrite=False, verbose=verbose)
 
+    processing = None
+    proc_history = raw.info['proc_history']
+    if proc_history != []:
+        sss = False
+        tsss = False
+        for ph in proc_history:
+            max_info = ph.get('max_info')
+            if max_info:
+                if (max_info['sss_info'] != dict() or
+                        max_info['sss_ctc'] != dict() or
+                        max_info['sss_cal'] != dict()):
+                    sss = True
+                if max_info['max_st'] != dict():
+                    tsss = True
+        if tsss:
+            processing = 'tsss'
```

Comment:
I would say you don't need this extra step. You should do this already in the loop.